initfiles/bin/init_thor

```diff
@@ -100,14 +100,13 @@ kill_slaves()
     if [[ "$localthor" = "true" ]]; then
         $deploydir/init_thorslave stop localhost $THORMASTER $THORMASTERPORT $LOG_DIR $instancedir $deploydir $THORNAME $PATH_PRE $logredirect
     else
-        # we want to kill only slaves that have already been started in run_thor
-        if [[ -r $instancedir/uslaves.start ]]; then
-            nslaves=$(cat $instancedir/uslaves.start 2> /dev/null | wc -l)
-            $deploydir/frunssh $instancedir/uslaves.start "/bin/sh -c '$deploydir/init_thorslave stop %a $THORMASTER $THORMASTERPORT $LOG_DIR $instancedir $deploydir $THORNAME $PATH_PRE $logredirect'" -i:$SSHidentityfile -u:$SSHusername -pe:$SSHpassword -t:$SSHtimeout -a:$SSHretries -n:$nslaves 2>&1
-        fi
+        daliadmin server=$DALISERVER dfsgroup ${groupName} slaves
```

Comment:
I think I would use the uslaves.start file if it exists, otherwise do the daliadmin cmd to recreate it.